daft/__init__.py

```diff
@@ -68,11 +68,38 @@ class daft:
 # Daft top-level imports
 ###
 
+from daft.convert import (
+    from_arrow,
+    from_dask_dataframe,
+    from_pandas,
+    from_pydict,
+    from_pylist,
+    from_ray_dataset,
+)
 from daft.dataframe import DataFrame
 from daft.datatype import DataType
 from daft.expressions import col, lit
+from daft.io import from_glob_path, read_csv, read_json, read_parquet
 from daft.series import Series
 from daft.udf import udf
 from daft.viz import register_viz_hook
 
-__all__ = ["DataFrame", "col", "DataType", "lit", "Series", "register_viz_hook", "udf"]
+__all__ = [
+    "from_pylist",
+    "from_pydict",
+    "from_arrow",
+    "from_pandas",
+    "from_ray_dataset",
+    "from_dask_dataframe",
+    "from_glob_path",
```

Comment:
While we're at it - wonder what you think about `from_glob_path -> glob_files`?

I've had folks ask questions about whether this would actually read the files into the DataFrame as bytes, and maybe this is a little clearer that it doesn't?

Reply:
Hmm I actually think that `from_glob_paths` or something akin to that connotes that better. We should quickly discuss this today to see if we can establish a consensus, and I can open a follow-up if we want to tweak the name.